main_test.go

```diff
@@ -212,9 +213,17 @@ func Test_Main_SampleXML(t *testing.T) {
 		t.Error(err)
 	}
 
-	collectorPort, err := otelCollector.MappedPort(ctx, "4317/tcp")
+	var collectorPort nat.Port
+	err = retry.Do(func() error {
```

Comment:
Suggestion: Let's not fix this here with the retry, and wait for my bump of the Testcontainers library. Can you remove the import too?

Reply:
This was a mistake. I'm juggling a few branches and goofed a merge.